base/components/PropControl.jsx

```diff
@@ -192,6 +194,40 @@ const PropControl = ({className, ...props}) => {
 	// Hack: Checkbox has a different html layout :( -- handled below
 	const isCheck = PropControl.KControlType.ischeckbox(type); // || PropControl.KControlType.isradio(type);
 
+	const isPublished = () => {
+
+		const getCTYPES = (typeString) => {
```

Comment:
Hm -- we should avoid putting Portal specific constants into the generic PropControl. 

But: Is this needed? It looks like mostly a no-op that returns the input or null. 

Would `C.TYPES.has()` cover what you want?

Reply:
@winterstein right, seems like I could use has(). I will still need to catch some constants like USER tho, I will see if I can write what I need in C.js. Thanks!

Reply:
FYI - C.TYPES is defined in the shared CBase.js -- to ensure it always exists. Each of our apps then resets it to their own list in their C.js file.